.github/workflows/test.yml

```diff
@@ -9,35 +9,50 @@ on:
       - 'master'
 
 jobs:
-  build:
-    name: PHP ${{ matrix.php }} - Laravel ${{ matrix.laravel }}
+  test:
     runs-on: ubuntu-latest
+
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
-        php: [ '7.2', '7.3', '7.4' ]
-        laravel: [ '5.5.*', '5.6.*', '5.7.*', '5.8.*', '^6.0', '^7.0' ]
+        php: [ 7.4, 7.3 ]
+        laravel: [ 8.*, 7.* ]
```

Comment:
Laravel version 6 is supported until September 3rd, 2022. I would like to keep support for that.

Reply:
Okay, but the test failed and currently, I don’t have the time to give it a better look 🙂

Reply:
Is it okay if I do the modifications on your pr to be able to merge it?

Reply:
Sure, feel free to do your modifications 😁